src/Hydrator/EntityHydrator.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace GraphAware\Neo4j\OGM\Hydrator;
+
+use GraphAware\Common\Result\Record;
+use GraphAware\Common\Result\Result;
+use GraphAware\Common\Type\Node;
+use GraphAware\Neo4j\OGM\Common\Collection;
+use GraphAware\Neo4j\OGM\EntityManager;
+use GraphAware\Neo4j\OGM\Metadata\NodeEntityMetadata;
+
+class EntityHydrator
+{
+    /**
+     * @var EntityManager
+     */
+    protected $_em;
```

Comment:
Should be private and one should remove the underscore. Same with $_classMetadata